test-data/unit/check-generic-subtyping.test

```diff
@@ -99,8 +99,11 @@ class F: pass
 from typing import TypeVar, Generic
 t = TypeVar('t')
 class I(Generic[t]): pass
-class A(I[C], I[object]): pass # E: Duplicate base class "I"
+class A(I[C], I[object]): pass
 class C: pass
+[out]
+main:4: error: Duplicate base class "I"
+main:4: error: Cannot determine consistent method resolution order (MRO) for "A"
```

Comment:
Good idea. I'm still setting a dummy MRO, because the crash occurs later in the same phase for the same file, and blockers don't stop processing until the phase is done; but it avoid the consistent MRO error, which comes from a later phase.